src/index.jsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { createRoot } from 'react-dom/client';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from 'react-router-dom';
-
 import { store } from './app/store';
 import App from './App';
 import './index.css';
+import './i18n'
```

Comment:
Global imports must be on top, add an empty line and after that add your local imports below it.